### PR TITLE
ci: use consistent binary dest path naming

### PR DESCRIPTION
**Pull request by jamilbk**

### CODE REVIEW

**Comment by ReactorScram** [comment]
Waiting to see what the CI output looks like

**Review by ReactorScram — CHANGES_REQUESTED** [review]
Something went off here https://github.com/firezone/firezone/actions/runs/8822073323/job/24219388108#step:11:23

**Comment by jamilbk** [comment]
CI is green now 🤞

File: scripts/build/tauri-rename-ubuntu.sh [hunk 1]
```diff
@@ -7,25 +7,23 @@ ls ../target/release ../target/release/bundle/deb
 # Used for release artifact
 # In release mode the name comes from tauri.conf.json
 # Using a glob for the source, there will only be one exe and one deb anyway
-cp ../target/release/firezone "$BINARY_DEST_PATH"-amd64
-cp ../target/release/firezone-gui-client.dwp "$BINARY_DEST_PATH"-amd64.dwp
-cp ../target/release/bundle/deb/*_amd64.deb "$BINARY_DEST_PATH"_amd64.deb
```
Comment (ReactorScram) [inline scripts/build/tauri-rename-ubuntu.sh:12]: I think the deb having an underscore where the other ones had a hyphen was just a convention, and Debian won't actually care if we break that, so this should be okay.

**Review by ReactorScram — APPROVED** [review]
Looks like it might work. That's a lot more places to put versions in, though.

**Pull request merged**